src/tests/ftest/dfuse/bash.py

```diff
@@ -119,3 +178,48 @@ def test_bashcmd(self):
                 self.container.destroy()
             # destroy pool
             self.pool.destroy()
+
+    def test_bashcmd(self):
+        """
+
+        Test Description:
+            Purpose of this test is to mount different mount points of dfuse
+            for different container and pool sizes and perform basic bash
+            commands.
+
+        :avocado: tags=all,pr,daily_regression
+        :avocado: tags=hw,medium
+        :avocado: tags=dfuse
+        :avocado: tags=Cmd,test_bashcmd
+        """
+        self.run_bashcmd()
+
+    def test_bashcmd_ioil(self):
+        """
+
+        Test Description:
+            Purpose of this test is to mount different mount points of dfuse
+            for different container and pool sizes and perform basic bash
+            commands.
+
+        :avocado: tags=all,pr,daily_regression
+        :avocado: tags=hw,medium
+        :avocado: tags=dfuse
+        :avocado: tags=Cmd,test_bashcmd,test_bashcmd_ioil
+        """
+        self.run_bashcmd(il_lib="libioil.so")
+
+    def test_bashcmd_pil4dfs(self):
+        """
+
+        Test Description:
+            Purpose of this test is to mount different mount points of dfuse
+            for different container and pool sizes and perform basic bash
+            commands.
+
+        :avocado: tags=all
+        :avocado: tags=hw,medium
+        :avocado: tags=dfuse
+        :avocado: tags=Cmd,test_bashcmd,test_bashcmd_pil4dfs
```

Comment:
```suggestion
        :avocado: tags=all
        :avocado: tags=hw,medium
        :avocado: tags=dfuse,pil4dfs
        :avocado: tags=Cmd,test_bashcmd_pil4dfs
```

Reply:
Thanks a lot for your code! Fixed as you suggested.